dependencies/common/install-npm-dependencies

```diff
@@ -85,7 +85,7 @@ if [ ! -x "$(command -v yarn)" ]; then
   rm $YARN_INSTALL_SCRIPT
   INSTALL_PATH="${HOME}/.yarn/bin:${HOME}/.config/yarn/global/node_modules/.bin:${INSTALL_PATH}"
 else
-  YARN_VERSION=`PATH=${INSTALL_PATH} yarn --version`
+YARN_VERSION=`$(PATH="${INSTALL_PATH}" yarn --version)`
```

Comment:
```suggestion
YARN_VERSION=$(PATH="${INSTALL_PATH}" yarn --version)
```

I think you want to either use backticks or `$()` for shell expansion, not both.